arcgis-java-toolkit-examples/src/main/java/com/esri/arcgisruntime/toolkit/examples/FloorFilterExample.java

```diff
@@ -0,0 +1,179 @@
+    private final VBox settings;
+    private final BorderPane borderPane;
+
+    /**
+     * A constructor for the FloorFilterExample that demonstrates how to implement a FloorFilter for a MapView.
+     *
+     * A Tab is created for the view to display in the application and settings are configured for testing and demo
+     * purposes.
+     *
+     * @since 100.15.0
+     */
+    public FloorFilterExample() {
+        // configure mapview tab
+        ArcGISMap map = new ArcGISMap("https://www.arcgis.com/home/item.html?id=f133a698536f44c8884ad81f80b6cfc7");
+        mapView.setMap(map);
+        borderPane = new BorderPane();
+        // instantiate a FloorFilter passing in the MapView
+        floorFilter = new FloorFilter(mapView);
+        // configure the UI for displaying the FloorFilter and MapView
+        // here the FloorFilter is displayed in a BorderPane alongside the MapView
+        floorFilter.minHeightProperty().bind(borderPane.heightProperty());
+        borderPane.setLeft(floorFilter);
+        borderPane.setCenter(mapView);
+        Tab mapViewTab = ExampleUtils.createTab(borderPane, "Map");
+
+        // add the tab to the list
+        tabs.add(mapViewTab);
+
+        // configure settings options
+        settings = configureSettings();
+    }
+
+    /**
+     * Sets up the required UI elements for toggling properties and other settings relating to the Toolkit Component.
+     *
+     * For the FloorFilter this includes AutomaticSelectionMode, size and position.
+     *
+     * @return a VBox containing the settings
+     * @since 100.15.0
+     */
+    private VBox configureSettings() {
+        // define specific settings for the FloorFilter
+        List<Node> requiredSettings = new ArrayList<>();
+
+        // automatic selection mode
+        VBox autoSelectVBox = new VBox(5);
+        Label autoSelectLabel = new Label("Automatic selection mode:");
+        ComboBox<FloorFilter.AutomaticSelectionMode> autoSelectComboBox = new ComboBox<>();
+        autoSelectComboBox.getItems().addAll(FloorFilter.AutomaticSelectionMode.ALWAYS,
+                FloorFilter.AutomaticSelectionMode.ALWAYS_NON_CLEARING, FloorFilter.AutomaticSelectionMode.NEVER);
+        autoSelectComboBox.getSelectionModel().selectedItemProperty().addListener((observable, oldValue, newValue) -> {
+            floorFilter.setAutomaticSelectionMode(newValue);
+        });
+        autoSelectComboBox.getSelectionModel().select(FloorFilter.AutomaticSelectionMode.ALWAYS);
+        autoSelectVBox.getChildren().addAll(autoSelectLabel, autoSelectComboBox);
+        requiredSettings.add(autoSelectVBox);
+
+        // Layout settings
+        TitledPane layoutTitledPane = new TitledPane();
+        layoutTitledPane.setExpanded(false);
+        layoutTitledPane.setText("Layout settings");
+        VBox layoutVBox = new VBox(5);
+        layoutTitledPane.setContent(layoutVBox);
+        // resize
+        Label sizeLabel = new Label("Resize:");
```

Comment:
it feels like some of this UI code could be naturally moved out/hidden into an fxml file: was there a reason you didn't opt for this for this class?

Reply:
Just time with initial implementation, I would say this is incomplete too as there's more settings that could be added for the FloorFilter. Wanted to add min. 2 Examples to begin with so hope future work from someone can add Examples for the rest of the tools and enhance the initial ones.